pkg/cmd/pulumi/destroy.go

```diff
@@ -155,7 +151,12 @@ func newDestroyCmd() *cobra.Command {
 					return result.FromError(err)
 				}
 
-				return runDeployment(ctx, opts.Display, apitype.Destroy, stackName, args[0], remoteArgs)
+				url := ""
```

Comment:
Nit: We generally use this syntax for a variable with default value (applies throughout)

```suggestion
				var url string
```